src/components/ResultsBody/Wrapper/index.jsx

```diff
@@ -0,0 +1,72 @@
+import styles from "./wrapper.module.css";
+import {
+  Autocomplete,
+  Box,
+  Button,
+  Icon,
+  Link,
+  MenuItem,
+  TextField,
+} from "@mui/material";
+import { PaginatedList } from "../PaginatedList/index.jsx";
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+
+export function Wrapper({ arrayOfSMLinks }) {
```

Comment:
Wrapper is to generic name for such a specialized component

Reply:
done